job_watcher/jobwatcher/job_watcher.py

```diff
@@ -5,17 +5,23 @@
 import datetime
 import json
 import os
+import sys
+import time
+from http import HTTPStatus
 from json import JSONDecodeError
 from time import sleep
-from typing import Any
+from typing import Any, Literal, cast
 
-from db.data_models import State
-from db.utils.db_updater import DBUpdater
+import requests
 from kubernetes import client  # type: ignore[import-untyped]
 from kubernetes.client import V1ContainerStatus, V1Job, V1Pod  # type: ignore[import-untyped]
 
 from jobwatcher.utils import logger
 
+StateString = Literal["SUCCESSFUL", "UNSUCCESSFUL", "ERROR", "NOT_STARTED"]
+FIA_API_HOST = os.environ.get("FIA_API", "fia-api-service.fia.svc.cluster.local:80")
+FIA_API_API_KEY = os.environ.get("FIA_API_API_KEY", "")
```

Comment:
```suggestion
FIA_API_API_KEY = os.environ.get("FIA_API_API_KEY")
```
Perhaps we should throw if FIA_API_API_KEY is undefined?

Reply:
Good idea

Reply:
Actually, this will throw anyway. If the API calls fails 3 times, the sys exit will trigger.

Reply:
It's more to do with defending against poor deployments